### PR TITLE
Feature/fe 84 develop news page

### PR DESCRIPTION
**Pull request by Yurii19**

### CODE REVIEW

**Comment by Yurii19** [comment]
It seems like home page does not work

**Comment by Yurii19** [comment]
News  should store in the NGXS store

File: src/app/main/pages/map/map.component.scss
Comment (Yurii19) [inline src/app/main/pages/map/map.component.scss:1]: This file is needed

File: src/app/main/pages/news/news.component.ts [hunk 1]
```diff
@@ -1,13 +1,45 @@
...
+  ],
+  providers: [{ provide: ArticlesService, useClass: ArticlesService }]
 })
-export class NewsComponent {
 
+export class NewsComponent implements OnInit {
+public readonly categories: string[] = ['Усі', 'Зустрічі', 'Лекції', 'Публікації', 'Майстер-класи', 'Концерти', 'Конференції'];
+public articles!: Article[];
+private readonly articlesService = inject(ArticlesService);
+private readonly destroyRef = inject(DestroyRef);
```
Comment (Yurii19) [inline src/app/main/pages/news/news.component.ts]: Should store data in ngxs storage, similar to expeditions.

File: .vscode/settings.json
Comment (Yurii19) [inline .vscode/settings.json:1]: I'm not sure if this change is appropriate.

File: db.json
Comment (Yurii19) [inline db.json:1]: Please check if this data is being used, if not, please delete the file

File: src/app/main/pages/news/news.component.ts [hunk 2]
```diff
@@ -1,13 +1,45 @@
...
+  templateUrl: './news.component.html',
+  styleUrls: ['./news.component.scss'],
+  standalone: true,
+  imports: [
+    TranslateModule,
+    RouterOutlet,
+    RouterLink,
+    ArticleItemComponent,
+    FilterComponent,
+    NgForOf,
```
Comment (Yurii19) [inline src/app/main/pages/news/news.component.ts]: Import CommonModule instead NgForOf

File: src/typography/_colors.scss [hunk 3]
```diff
@@ -6,7 +6,7 @@
   --bg-color-yellow: rgba(255, 240, 104, 100%);
   --bg-color-transparent-yellow: rgba(254, 254, 254, 10%);
   --bg-color-footer: rgba(117, 79, 49, 100%);
-  --text-color-primary: rgba(19, 19, 19, 100%);
+  --text-color-primary: #131313;
```
Comment (Yurii19) [inline src/typography/_colors.scss]: Seems that is unnecessary change

File: .vscode/settings.json
Comment (shytuleva24) [reply .vscode/settings.json:1]: back to square one

File: db.json
Comment (shytuleva24) [reply db.json:1]: file deleted

File: src/app/main/pages/news/news.component.ts (hunk 2 above)
Comment (shytuleva24) [reply src/app/main/pages/news/news.component.ts]: Import CommonModule instead NgForOf

File: src/typography/_colors.scss (hunk 3 above)
Comment (shytuleva24) [reply src/typography/_colors.scss]: back to square one.

File: src/app/main/pages/news/news.component.ts (hunk 1 above)
Comment (shytuleva24) [reply src/app/main/pages/news/news.component.ts]: is now storing data in the ngxs repository, similar to expeditions

File: src/app/main/pages/map/map.component.scss
Comment (shytuleva24) [reply src/app/main/pages/map/map.component.scss:1]: File returned

File: package.json [hunk 4]
```diff
@@ -28,17 +28,18 @@
     "@angular/router": "^16.1.0",
     "@ngx-translate/core": "^15.0.0",
     "@ngx-translate/http-loader": "^8.0.0",
+    "@ngxs/devtools-plugin": "^3.8.1",
```
Comment (Yurii19) [inline package.json]: Is it necessary ? i think you should uninstal this plugin

File: src/app/main/pages/news/news.component.ts [hunk 5]
```diff
@@ -1,13 +1,81 @@
...
+  providers: [{provide: ArticlesService, useClass: ArticlesService}]
 })
-export class NewsComponent {
 
+export class NewsComponent implements OnDestroy {
+  @Select(NewsState.getArticlesList) setArticles$?: Observable<Article[]>;
+  categories: newsCategories[] = Object.values(newsCategories);
+  public articles!: Article[];
+  public filteredArticle!: Article[];
+  private articlesSubscription?: Subscription;
```
Comment (Yurii19) [inline src/app/main/pages/news/news.component.ts]: It seems like unused variables ?

Comment (shytuleva24) [reply src/app/main/pages/news/news.component.ts]: all the variables are used

File: package.json (hunk 4 above)
Comment (shytuleva24) [reply package.json]: uninstal
